src/widget-components/conservation-hotspots/component.js

```diff
@@ -25,16 +26,15 @@ function ConservationHotspots({
   isCollapsed,
   slug,
   name,
+  ui: scope,
+  setUi,
```

Comment:
Missing in props validation

Reply:
scope: PropTypes.string,
  setUi: PropTypes.func

  scope: null,
  setUi: () => {}